Mathlib/SetTheory/Ordinal/Principal.lean

```diff
@@ -42,41 +41,45 @@ namespace Ordinal
 ordinals less than it is closed under that operation. In standard mathematical usage, this term is
 almost exclusively used for additive and multiplicative principal ordinals.
 
-For simplicity, we break usual convention and regard 0 as principal. -/
+For simplicity, we break usual convention and regard `0` as principal. -/
 def Principal (op : Ordinal → Ordinal → Ordinal) (o : Ordinal) : Prop :=
   ∀ ⦃a b⦄, a < o → b < o → op a b < o
 
-theorem principal_iff_principal_swap {op : Ordinal → Ordinal → Ordinal} {o : Ordinal} :
-    Principal op o ↔ Principal (Function.swap op) o := by
+theorem principal_swap_iff {o : Ordinal} : Principal (Function.swap op) o ↔ Principal op o := by
```

Comment:
You're flipping and renaming a lemma. Can you add a deprecated alias for the old lemma?

Reply:
The old lemma (and actually this new one too) weren't used anywhere. My reason for flipping it is that iff statements like these should have the LHS be simpler.

But sure, I can still add the label.